client/src/components/columns.jsx

```diff
@@ -1,55 +1,38 @@
 import React, { Component } from "react";
 import axios from "axios";
-let arrhappy = [];
-let arrsad = [];
-const api = "http://localhost:3001/sites";
 
 class Columns extends Component {
   state = {
-    data: [{ id: 0, name: "", urls: ["", ""] }],
     happy: [],
     sad: [],
     faking: []
   };
   componentDidMount() {
     this.getDataFromDb();
-    this.parseData(this.state.data);
   }
 
   getDataFromDb = () => {
-    axios.get(api).then(result => {
-      console.log(result.data);
-      let variable = [];
-      variable = result.data;
-      this.setState({ data: variable });
+    axios.get("/sites/check").then(result => {
+      this.updateStatus(result.data);
     });
   };
 
-  parseData = data => {
-    data.map(site => {
-      this.checkStatus(site);
-    });
-  };
 
-  checkStatus(site) {
-    site.urls.map(url =>
-      axios.get(url).then(response => {
```

Comment:
it's not possible to do request to external service from react - because of [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS). 
The easiest way to call other servers - is to use backend - there are no such limitations.